src/Footer.svelte

```diff
@@ -0,0 +1,55 @@
+<script>
+  import { mode, tasks, activeCount } from "./stores";
+
+  export let alertsOn;
+
+  // function to clear all tasks
+  function clearAllTasks() {
+    if ($tasks.length !== 0) {
+      tasks.set([]);
+
+      if (alertsOn) {
+        alert("All done! Time for a drink! 🍸");
+      }
+    }
+  }
+</script>
+
+<footer>
+  <span>
+    {$activeCount}
+    {$activeCount === 1 ? "task" : "tasks"} remaining
+  </span>
+  <span>|</span>
+  <span>
+    Showing {$mode} tasks
+  </span>
+  <span>|</span>
+  <button on:click={() => mode.set("all")}>All</button>
+  <span>|</span>
+  <button on:click={() => mode.set("active")}>Active</button>
+  <span>|</span>
+  <button on:click={() => mode.set("completed")}>Completed</button>
+  <span>|</span>
+  <button on:click={clearAllTasks}>Clear All Tasks</button>
+  <span>|</span>
+  <button on:click={() => (alertsOn = !alertsOn)}>Toggle Alerts</button>
+</footer>
+
+<style>
+  footer {
+    position: fixed;
+    inset: auto 0 0 0;
```

Comment:
[`inset`](https://developer.mozilla.org/en-US/docs/Web/CSS/inset) is a shortcut for setting `top`, `left`, `bottom`, and `right` separately.